src/components/ProfileDetail/UserProfile/UserProfile.tsx

```diff
@@ -5,7 +5,7 @@ import { selectUser } from '@store/slices/user';
 import Button from '@mui/material/Button';
 import Avatar from '@mui/material/Avatar';
```

Comment:
p4;
뭉쳐도 좋을 것 같습니다. :) 

```suggestion
import {Button,Avatar} from '@mui/material';
```